login/service_whitebox_test.go

```diff
@@ -156,8 +156,7 @@ func TestEncodeTokenOK(t *testing.T) {
 	refreshToken := "refreshToken%@!/\\&?"
 	tokenType := "tokenType%@!/\\&?"
 	expiresIn := 1800
-	var refreshExpiresIn float64
-	refreshExpiresIn = 2.59e6
+	refreshExpiresIn := 2.59e6
```

Comment:
done in https://github.com/fabric8-services/fabric8-wit/pull/1489/commits/11a85853665905840341dc5a1df3545d84c0eaea#diff-ce0aa7f266a3f01ad173ef2e98624125R159